server/services/overviewPageService.ts

```diff
@@ -20,6 +21,7 @@ export default class OverviewPageService {
     const nonAssociations = await this.getNonAssociations(prisonerNumber)
     const miniSummaryGroupA = await this.getMiniSummaryGroupA(prisonerNumber, bookingId)
     const miniSummaryGroupB = await this.getMiniSummaryGroupB(currentIncentive, bookingId)
+    const personalDetails = this.getPersonalDetails(prisonerData)
```

Comment:
We could do with a test around this to make sure it's properly creating the return value correctly as it's doing some data manipulation/mapping